faqs/galaxy/dataset_metadata_missing.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Missing or Incomplete Dataset metadata 
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq         
+---
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon
+- Tools error when using a previously successfully used specific dataset
+- Tools error with a message that ends with: ``OSError: [Errno 2] No such file or directory.``
+
+Solution:
+Click on the the dataset's {% icon galaxy-pencil %} ✏️ to reach the _Edit Attributes_ forms and click either:
```

Comment:
The formatting will not appear very nicely here. You either need to add a list (or numbered list) or have a blank line in between.